cmd/frontend/internal/cli/serve_cmd.go

```diff
@@ -112,14 +112,12 @@ func InitDB() error {
 }
 
 // Main is the main entrypoint for the frontend server program.
-func Main(githubWebhook, bitbucketServerWebhook http.Handler) error {
+func Main(enterpriseSetupHook func()) error {
```

Comment:
How about getting rid of the global variables and have the webhook handlers be returned here?

```suggestion
func Main(enterpriseSetupHook func() []http.Handler) error {
```

I think that would make the code easier to understand and less error-prone when later moving things around.